migrations/env.py

```diff
@@ -31,30 +31,25 @@
 # Define and register the stored procedures for VA Profile integration.
 #######################################################################
 
```

Comment:
This function is outdated.  I'm assuming it is replaced by Maham's stored function PR